.ci/containers/downstream-builder/generate_downstream.sh

```diff
@@ -160,6 +160,9 @@ else
         fi
         pushd ../
         make tpgtools OUTPUT_PATH=$LOCAL_PATH VERSION=$VERSION
+        pushd ./tools/breaking-change-detector
+        go run . -docs -providerFolder="${LOCAL_PATH}/.github/"
```

Comment:
Is there any danger that this will break generation if breaking-change-detector doesn't exist? Or is it run with the merged branch on main so it will always exist even for older PRs?

Reply:
For very old prs. yes it will break. Unfortunate, but thats just how it is.